scss/_patterns_buttons.scss

```diff
@@ -149,5 +149,9 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
         margin-right: -#{$sph-inner--small};
       }
     }
+
+    & .p-icon--success {
```

Comment:
Adding this here makes it work on all the buttons, not just positive/negative/brand.
Now if you put success icon in base button it's invisible (white on white background).

Reply:
Maybe we can create custom placeholder :

```scss
%vf-button-white-success-icon {
   & .p-icon--success {
      @include vf-icon-success($color-x-light, $color-transparent);
   }
}
```
And then `@extend` it where `--positive`, `--negative`, `--brand` buttons are defined?

Reply:
good idea - I'd added that to the classes directly, I've just changed it to a placeholder.